webapp/publisher/snaps/views.py

```diff
@@ -896,6 +896,7 @@ def get_publicise(snap_name):
         "snap_title": snap_details["title"],
         "snap_id": snap_details["snap_id"],
         "available": available_languages,
+        "download_version": "v1.1",
```

Comment:
Too bad we would need to update this manually every time. Hopefully not to often.

There seems to be a way to get latest version from API, but probably not worth implementing at this point.

Reply:
Yeah I thought about that, I think we might update more frequently in the coming months, but updates will probably slow